msg/estimator_event_flags.msg

```diff
@@ -0,0 +1,32 @@
+uint64 timestamp                          # time since system start (microseconds)
+uint64 timestamp_sample                   # the timestamp of the raw data (microseconds)
+
+# information events
+uint32 information_event_changes	# number of information event changes
+bool gps_checks_passed			#  0 - true when gps quality checks are passing passed
+bool reset_vel_to_gps			#  1 - true when the velocity states are reset to the gps measurement
```

Comment:
I'm still wondering if some of these flags might be useful as counts to both eliminate the potential confusion if a log drop out occurs and make it clear from inspection if certain resets are occurring frequently (eg `listener estimator_event_flags`).